lib/index.js

```diff
@@ -67,6 +67,14 @@ function preprocessor(css, options, filename) {
     return settings ? plugin(settings) : plugin;
   });
 
+  // autoprefixer should always be the last plugin
+  plugins.push(require('autoprefixer')(options.autoprefixer));
```

Comment:
I think we'd still need `postcss-reporter` at the end, otherwise errors from `autoprefixer` will be lost.

I always find it a bit neater to keep the requires at the top too.